ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkReader.java

```diff
@@ -79,7 +79,7 @@ private static AmplException createWrongNumberOfColumnException(int expected, in
         return new AmplException("Wrong number of columns " + actual + ", expected " + expected);
     }
 
-    private static List<String> parseExceptIfBetweenQuotes(String str) {
+    protected static List<String> parseExceptIfBetweenQuotes(String str) {
         List<String> tokens = new ArrayList<>();
         Matcher m = PATTERN.matcher(str);
         while (m.find()) {
```

Comment:
As discussed, this while can be avoided with a better pattern. And all single quotes are replaced, but not double quotes. But it's ok for this PR!